base.js

```diff
@@ -211,10 +211,13 @@ const baseConfig = {
       // Cypress config
       files: [`**/cypress/**/*.{${allExtensions}}`],
       env: {
-        'cypress/globals': true,
+        '@finsit/cypress/globals': true,
```

Comment:
Would it be worth mentioning the reasoning in the PR here in some way?

Reply:
Was this meant to be commented on the other thread?

Reply:
No. I was thinking like:
```suggestion
        // eslint-plugin-cypress doesn't support ESLint v8. Use fork by `@finsit` until this is solved. https://github.com/cypress-io/eslint-plugin-cypress/issues/89
        '@finsit/cypress/globals': true,
```

Reply:
Oh, but "in the PR" you meant "in the code", not the PR description. Got it.

Reply:
Well actually this could probably be on the "extends" two lines down